pennylane/transforms/zx/converter.py

```diff
@@ -331,16 +344,45 @@ def processing_fn(res):
         )
 
         expanded_operations = []
-
-        # Define specific decompositions
         for op in mapped_tape.operations:
-            if op.name == "RY":
-                theta = op.data[0]
+            if op.name == "Toffoli":
+                # Explicit list of ops given by op.decomposition, replacing Adjoint(T) with PhaseShift(-π/4)
+                decomp = [
+                    qml.Hadamard(wires=op.wires[2]),
```

Comment:
Can we extract these into helper functions like `_toffoli_clifford_t_decomp`?

Reply:
Yep right, makes sense!